### DISCUSSION TITLE
Switch to Bun?

### DISCUSSION THREAD

**Discussion by zAlweNy26** [zAlweNy26/ts-cat#2] [Ideas]
Switching to Bun could be a nice idea to improve performance, but we should be aware of possible incompatibilities with libraries.

**Comment by valentimarco** [comment]
YES

**Comment by zAlweNy26** [comment]
I made a branch for the migration: [here](https://github.com/zAlweNy26/ts-cat/tree/bun-refactor)
Everything seems fine and even more performant, I'm just having some problems with langchain types.

**Reply by zAlweNy26** [reply]
Nevermind, I fixed those type errors, just a lockfile issue.
